pkg/bpf/fake_mac.go

```diff
@@ -0,0 +1,293 @@
+//go:build darwin
+// +build darwin
```

Comment:
Kepler won't run on osx, so IMHO, we should spend time on it. WDYT?

Reply:
well, just because I am a mac user.
as a mac user, when I notice/make some nit fix. 
I want to test those locally.
so, I made some build tag on files to make it happen.
any other suggestions?

Reply:
I am completely fine with supporting darwin as long as we don't block it in CI as OSX isn't a supported platform. 
I also feel it is best to focus on linux as the platform kepler support and hence all development on OSX is expected to use linux box ([lima](https://github.com/lima-vm/lima) may work well)

Reply:
yep, I agree currently darwin is not major platform to run kepler.
so, @sthaha , can we make an agreement as following:
> kepler currently focus on linux platform.
> for other platforms, to make kepler is easy for anyone contributes from any platform, we are welcome any benefits(PRs) for kepler including parts as compilable on other platform. 
> before the specific platform is supported, we just running CI on linux as PR merge standard and official support.

as instance, this PR, increasing test coverage on config package(benefits), with parts as compilable on darwin.